docs/interpreter/jdbc.md

```diff
@@ -865,11 +865,7 @@ Dependencies
     <th>Excludes</th>
   </tr>
   <tr>
-    <td>org.apache.kyuubi:kyuubi-hive-jdbc-shaded:1.6.1-incubating</td>
-    <td></td>
-  </tr>
-  <tr>
-    <td>org.apache.hive:hive-jdbc:3.1.2</td>
+    <td>org.apache.kyuubi:kyuubi-hive-jdbc-shaded:1.9.0</td>
```

Comment:
previous might be a mistake, the `kyuubi-hive-jdbc-shaded` is an all-in-one all, we don't need the `hive-jdbc`